### DISCUSSION TITLE
get_pvgis_hourly

### DISCUSSION THREAD

**Discussion by jesuspolo** [pvlib/pvlib-python#1890] [Q&A] [answered]
I was wondering why this function only allows to retrieve hourly data from 2005 to 2016. The web application of pvgis allows to get data from SARAH2, for instance, from 2005 to 2020. Thanks a lot for any clarification

**Comment by kandersolar** [comment, marked as answer]
I think the most recent data is only available in PVGIS version 5.2.  The API that pvlib accesses is still serving 5.1.  If you specify the 5.2 endpoint like this, you can get 2020 data:

```python
import pvlib
df, params, meta = pvlib.iotools.get_pvgis_hourly(45, 8, raddatabase='PVGIS-SARAH2', url='https://re.jrc.ec.europa.eu/api/v5_2/')
```

More info: 
- https://joint-research-centre.ec.europa.eu/photovoltaic-geographical-information-system-pvgis/pvgis-releases/pvgis-52_en
- #1415

**Reply by jesuspolo** [reply]
Thanks so much!!!! This is very helpful. Great!!
